src/utils/filters.ts

```
@@ -10,7 +10,8 @@
 
 export const fileCacheAgentFilter: AgentFilterFunction = async (context, next) => {
   const { namedInputs } = context;
-  const { file, force, mulmoContext, index, sessionType } = namedInputs;
+  const { file, cache } = namedInputs;
```

ESLint

[error] Remove this useless assignment to variable "cache".

(sonarjs/no-dead-store)